test/scenarios/el93-src@standard-suite.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Sourced from cleanup_scenario.sh and uses functions defined there.
```

Comment:
```suggestion
# Sourced from scenario.sh and uses functions defined there.
```

Reply:
Well, I just copied the el92 file but I'll update that ;)